tests/api/holdings/test_patterns.py

```diff
@@ -142,20 +142,22 @@ def test_receive_regular_issue_api(
     # librarian of another library are not authoritzed to receive issues
     # for another library.
     login_user_via_session(client, librarian_fully_no_email.user)
-    res, data = postdata(
-        client,
-        'api_holding.receive_regular_issue',
-        url_data=dict(holding_pid=holding.pid)
-    )
-    assert res.status_code == 401
+    with pytest.raises(Exception):
+        res, data = postdata(
+            client,
+            'api_holding.receive_regular_issue',
+            url_data=dict(holding_pid=holding.pid)
+        )
+        assert res.status_code == 401
```

Comment:
Are you sure the function return à 401 ? 
As I read the test, the `with` statement is validated because `postdata` raise an `Exception` and the status_code isn't check. For me it's not the good. The API call to 'receive_regular_issue' should not raise any Exception and ony return 401 (So the previous test is correct)